libs/dh/metering-point/feature-overview/src/components/measurement/dh-measurements-month.component.ts

```diff
@@ -144,7 +144,9 @@ export class DhMeasurementsMonthComponent {
   private fb = inject(NonNullableFormBuilder);
   private transloco = inject(TranslocoService);
   private sum = computed(() =>
-    this.formatNumber(this.measurements().reduce((acc, x) => acc + x.quantity, 0))
+    this.formatNumber(
+      this.measurements().reduce((acc, x) => Number(acc) + Number(x.quantity ?? 0), 0)
```

Comment:
suggestion: let's change some of the mock data to reflect the fact that the value can be `number | string | null | undefined`